docs/index.md

```diff
@@ -36,6 +36,18 @@ RCS is developed and supported by UAB IT's Research Computing Group. We are deve
 
 RCS is an out growth of the UABgrid pilot, launched in September 2007 which has focused on demonstrating the utility of unlimited analysis, storage, and application for research. RCS is built on the same technology foundations used by major cloud vendors and decades of distributed systems computing research, technology that powered the last ten years of large scale systems serving prominent national and international initiatives like the [Open Science Grid](https://osg-htc.org/), [XSEDE](https://www.xsede.org/), the [LHC Computing Grid](https://wlcg.web.cern.ch/), and [NCIP](https://datascience.cancer.gov/).
 
+## Follow up ServiceNow Tickets
```

Comment:
Maybe this should be "How Can I Follow up on ServiceNow Tickets?"

At the very we need "Following up on" instead of just "Follow up", in this case.